### PR TITLE
Feature/teaser adjustments

### PR DESCRIPTION
**Pull request by tuj**
Added price to activity and teaser.

### CODE REVIEW

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 1]
```diff
@@ -147,6 +147,16 @@ function genlyd_aarhus_preprocess_node__activity(&$variables) {
   ];
   $mailto = implode("", $mailtoParts);
 
+
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Two new lines?

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 2]
```diff
@@ -147,6 +147,16 @@ function genlyd_aarhus_preprocess_node__activity(&$variables) {
...
+  // Set price text.
+  $price = $node->field_price->value;
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Another way could be this, but it's more a personal taste....

```
$priceRaw = $node->field_price->value;
$price = \Drupal::translation()->translate('Free');
if (isset($priceRaw) || $priceRaw > 0) {
   $price = \Drupal::translation()->translate(':price kr.', [ ':price' => $priceRaw ]);
}
```

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 3]
```diff
@@ -225,6 +232,15 @@ function genlyd_aarhus_preprocess_node__activity__teaser(&$variables) {
     $imageUrl = ImageStyle::load('activity_teaser')->buildUrl($image);
   }
 
+  // Set price text.
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: This is the same code as above in "genlyd_aarhus_preprocess_node__activity" and it's in the same file, so maybe a helper function ?

**Review by cableman — CHANGES_REQUESTED** [review]

**Review by cableman — APPROVED** [review]

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme (hunk 2 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Changed to private function

**Pull request merged**